docs/manual/advanced-features.tex

```diff
@@ -1246,20 +1247,12 @@
 
 
 \textbf{Limitations:}
-The following Java expressions may not currently be written:
-% The Checker Framework is best at reasoning about Java expressions that
-% are variable references, but these expressions are not.
-\begin{itemize}
-\item String concatenation expressions.
-\item Mathematical operators (plus, minus, division, ...).
-\item Comparisons (equality, less than, etc.).
-\end{itemize}
-
-Additionally, it is not possible to write
-quantification over all array components (e.g. to express that all
+It is not possible to write a
```

Comment:
Aren't at least comparisons still forbidden? Or is this partially addressing #3474?

Reply:
Binary operations are possible, so I'm not positive that comparisons are explicitly forbidden.  Since part of the list became out of date, I removed it so that the rest does not get out of date.

Reply:
Ok, so let's leave it for #3474 to add proper restrictions again.